package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "wavefront-sdk-javascript",
+  "version": "1.0.0",
+  "description": "Wavefront Core Javascript SDK",
+  "main": "dist/index.umd.js",
+  "scripts": {
+    "prebuild": "rimraf dist",
+    "build": "rollup -c",
+    "start": "node -r @babel/preset-env -r @babel/register -r regenerator-runtime example.js <server> <token>",
```

Comment:
you don't need to put <server> or <token> in here, npm will pass the rest of the command line into node